examples/boilerplate.py

```diff
@@ -0,0 +1,24 @@
+import pygame
```

Comment:
You should be importing `pygame` as `pg` here to keep consistency with the other examples.